scan-manager/docs/setup.md

```diff
@@ -26,9 +26,9 @@
 - Default `config.yaml` is provided inside `setup_artifacts/deployment` directory for initial setup
 - [Add](#adding-templates) corresponding templates for the given model types and base-image
 - Make sure to update the `license.txt` file for h2o-mojo in the `setup_artifacts/files` directory (when working with h2o-mojo models)
-- Run the `sh upload_artifact.sh <AWS_BUCKET>` script from the `setup_artifacts` directory   
-  to make the updated artifacts available for Scan Manager to get started
-- `AWS_BUCKET` is the same bucket configured when installing Scan Manager
+- Run the `bash upload_artifact.sh <END_POINT> <ACCESS_KEY> <SECRET_KEY> <BUCKET_NAME>` script from the `setup_artifacts` directory   
+  to make the updated artifacts available for Scan Manager to get started[Minio Client should be installed before running sh upload_artifact.sh]
```

Comment:
@mchutani-CS Pre-requisite has been added here, Are you expecting it to be somewhere else?

Reply:
No, but I'm expecting to have `mc` as a pre-requisite in this list (and even others if I missed something)

Reply:
This should have the pre-reqs to run this script.